lib/broadway_cloud_pub_sub/pull_client.ex

```diff
@@ -36,10 +36,24 @@ defmodule BroadwayCloudPubSub.PullClient do
   def receive_messages(demand, ack_builder, config) do
     max_messages = min(demand, config.max_number_of_messages)
 
-    config
-    |> execute(:pull, %{"maxMessages" => max_messages})
-    |> handle_response(:receive_messages)
-    |> wrap_received_messages(ack_builder)
+    :telemetry.span(
+      [:broadway_cloud_pub_sub, :pull_client, :receive_messages],
+      %{
+        max_messages: max_messages,
+        demand: demand,
+        topology_name: config.broadway[:name]
```

Comment:
To align with SQS:

```suggestion
        name: config.broadway[:name]
```

Reply:
Shouldn't this be `topology_name` to match Broadway's own telemetry? `name` seems confusing since it's the name of the topology but called `name` but emitted by the `PullClient`